### ISSUE TITLE
"dd" function blank response in 8.32.1

### ISSUE THREAD

**Issue by johnwedgbury** [laravel/framework#36535]
- Laravel Version: 8.32.1
- PHP Version: 7.4.10
- Database Driver & Version: MySQL 5.7.28

### Description:

Using `dd(` returns a blank response. This seems to be affected at least as far back as `8.30.0`

This works correctly when rolling back to `8.12.0`.

### Steps To Reproduce:

Controller
```
    /**
     * Update the specified resource in storage.
     *
     * @param  \Illuminate\Http\Request  $request
     * @param  \App\Customer  $customer
     * @return \Illuminate\Http\Response
     */
    public function update(Request $request, Customer $customer)
    {
        dd($request->all());
    }
```

Routes (`api.php`):
```
Route::resource('customers', CustomerController::class)->only(['update']);
```

Response is 200 OK but no data.

Rolling back to version `8.12.0` gives the expected result:
```
array:10 [
  "email" => "xxxxx@gmail.com"
  "title" => "xxxx"
  "first_name" => "xxxx"
  "surname" => "xxxx"
  "address_line_1" => "xxxx"
  "town" => "xxxx"
  "postcode" => "xxxx"
]
```

**Comment by stryaponoff** [comment]
I can confirm that issue - it affects my project too

**Comment by Mezzair** [comment]
Confirmed here. I don't think the `$request->all()` payload is even available.

**Comment by it-can** [comment]
What does var_dump show?

**Comment by elnur-ibr** [comment]
I think there is problem with `symfony/var-dumper`. Temporary solution is to depreciate it to 5.1.

**Comment by LucaRed** [comment]
The bug was introduced in `v8.29.0`.

As of today (2021-03-10) the latest working version is `v8.28.1`.

**Comment by taylorotwell** [comment]
Working fine for me. Any bug in dd() is likely to be a Symfony VarDumper issue.

**Comment by omarhen** [comment]
> Working fine for me. Any bug in dd() is likely to be a Symfony VarDumper issue.

 problem solved by downgrade Laravel to v8.28.1 ,  2 updates (Laravel ,Telescope)  , 
no changes for Symfony

![image](https://user-images.githubusercontent.com/5448144/110722237-6256ea80-822b-11eb-8b4d-ae9edcb689a3.png)

**Comment by driesvints** [comment]
This was caused by a change in Telescope. We tagged v4.4.6 that fixes this.
